src/runtime/auto-prompt-manager.js

```diff
@@ -452,6 +467,54 @@ export class AutoPromptManager {
     }
   }
 
+  /**
+   * Returns which type of prompt to display based on the type specified,
+   * the viewport width, and whether the disableDesktopMiniprompt experiment
+   * is enabled.
+   *
+   * If the disableDesktopMiniprompt experiment is enabled and the desktop is
+   * wider than 480px then the miniprompt type will be substituted for the large
+   * prompt. The miniprompt will be returned as is in all other cases.
```

Comment:
The fallback behavior is not specific to miniprompts, it is true regardless of the type of input prompt type.